classes/04class/exercises/c04-iac02/zarajoy/variables.tf

```diff
@@ -0,0 +1,57 @@
+#variable declaration
+variable "project-name" {
+  	type    = string
+   	description = "project name"
+  	default = "c04-iac02"
+}
+
+variable "vpcCIDR" {
+    description = "VPC network CIDR"
+    type = string	
+}
+
+variable "instanceTenancy" {
+    type    = list(string)
+    description = "defult or dedicated"
+    default = ["default","dedicated"]
```

Comment:
Unfortunately, terraform doesn't have enumerator so just use simple string here. You don't need a list since that you always pick the first item of the list.

Reply:
fixed :)